content/data-science/data-science.md

```diff
@@ -0,0 +1,61 @@
+---
+# This file is used to populate the hub page for the language whose folder it's in. Be sure to create a new version if you create a folder for a new language!
+
+Title: 'Data Science' # Required; this is the name that appears on the hub page for this language. Pay attention to capitalization and punctuation!
+Description: 'Data Science is a multidisciplinary field that includes activities like data management, data collection, data analysis, and data visualization.' # Required; ideally under 150 characters and starts with the name of the topic (used in search engine results and content previews)
+Codecademy Hub Page: 'https://www.codecademy.com/catalog/subject/data-science' # If codecademy.com doesn't have a hub page for this language, that's okay too. You can omit this field.
+CatalogContent: # Please use course/path landing page slugs, rather than linking to individual content items. If listing multiple items, please put the most relevant one first
+  - 'learn-data-science'
+  - 'paths/data-science-foundations'
+---
+
+**Data Science** is a multidisciplinary field of study that includes activities like data management, data collection, data analysis, and data visualization. It incorporates skills from a broad range of disciplines that include computer science, statistics, mathematics, and visual design.
+
+## Applications
+
+Since the beginning of the 21st century, data science has been used in generally every field of industry to extract insights from data that may be leveraged in business decision-making and product development. These applications are largely related to fields of study rooted in data science, including:
```

Comment:
```suggestion
Since the beginning of the 21st century, data science has been used in almost every field of industry to extract insights from data that may be leveraged in business decision-making and product development. These applications are largely related to fields of study rooted in data science, including:
```